test/i18n_test.py

```diff
@@ -55,11 +55,48 @@ def test_translates_random_english():
     assert expect == actual
 
 
-def test_translate_to_french():
-    i18n.install('fr_FR')
-    expect = ' commits avant'
-    actual = N_(' commits ago')
-    assert expect == actual
+def test_avoid_translation_git_jargon():
```

Comment:
If these strings should not be translated, wouldn't it be better to update the code to stop marking them as translatable, rather than testing that no translation is provided?

Reply:
I agree with you, I wanted it to remain consistent with previous commit [1343](https://github.com/git-cola/git-cola/pull/1343/commits) with similar changes in German.